configs/README.md

```diff
@@ -0,0 +1,264 @@
+# Maestro Config
+
+The following guide documents the options available in the `maestro.config` yaml file that is provided to maestro on startup.
```

Comment:
how is the file provided to maestro on startup?  a command line option?  hard-coded filename?